ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -17,17 +17,17 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/{{ eessi_version }}/compat/{{
 
 # How to build the prefix
 prefix_required_space: 15 GB
-prefix_snapshot_url: http://cvmfs-s0.eessi-hpc.org/snapshots
-prefix_snapshot_version: 20210322
-prefix_python_targets: python3_8
+prefix_snapshot_url: https://eessi-gentoo-snapshot.s3-eu-west-1.amazonaws.com
+prefix_snapshot_version: 20210607
+#prefix_python_targets: python3_9
```

Comment:
Should this line be removed?

I guess this is because Prefix doesn't use Python 2 by default anymore?

Reply:
Yep. I commented it for now, so that we can reuse it if we ever need to stick to a specific Python version again. But maybe we can document that somewhere instead?

Reply:
Maybe it's better to keep explicitly control over the Python version, since it's used for EasyBuild (and ReFrame).

If Python 3.10 comes out, I expect I'll have to make some minor tweaks to EasyBuild to avoid fallout, so sticking to Python 3.9 for now makes sense...

Reply:
(@amadio: correct me if I'm wrong) Gentoo supports having multiple Python versions next to each other, and by restricting this I think we can potentially run into issues with multiple packages requiring different/non-overlapping versions.
The tricky thing is that EasyBuild is not part of the compat layer, so in principle that could indeed also lead to issues. @amadio what would you recommend here?

I also removed this, by the way, as otherwise the Github Action breaks: the Docker image from awesomebytes hasn't been updated in a while and was built with an older version of the bootstrap script, which causes some issues when we run our playbook while sticking to Python 3.9.

Reply:
OK, it's fine then.
If we need an older version of Python for EasyBuild, we can always include it in our EESSI set?

Then we should clean up the commented out things imho...

Reply:
Done!